configure

```diff
@@ -81,36 +81,92 @@ int main() {
 }
 EOF
 
-# First, try R CMD SHLIB to see if R can already compile
-# things using OpenMP without any extra help from data.table
-"${R_HOME}/bin/R" CMD SHLIB test-omp.c >/dev/null 2>&1 || R_NO_OPENMP=1
-
-if [ "$R_NO_OPENMP" = "1" ]; then
-  # Compilation failed -- try forcing -fopenmp instead.
-  R_NO_OPENMP=0
-  "${CC}" "${CFLAGS}" -fopenmp test-omp.c || R_NO_OPENMP=1
-  # TODO: and then nothing seems to be done with this outcome
-else
-  echo "R CMD SHLIB supports OpenMP without any extra hint"
-fi
+detect_openmp () {
+
+  if [ "$(uname)" = "Linux" ]; then
```

Comment:
Maybe this should also handle other *nix platforms (other than macOS)?

Reply:
I think it's fine like this for now, a follow-up would be easy later if needed/requested